heat/core/statistics.py

```diff
@@ -1068,6 +1083,73 @@ def minimum(x1, x2, out=None):
     return lresult
 
 
+def __moment_w_axis(function, x, axis, elementwise_function, unbiased=None, Fischer=None):
+    # helper for calculating a statistical moment with a given axis
+    kwargs = {"dim": axis}
+    if unbiased:
+        kwargs["unbiased"] = unbiased
+    if Fischer:
+        kwargs["Fischer"] = Fischer
+
+    output_shape = list(x.shape)
+    if isinstance(axis, (list, tuple, dndarray.DNDarray, torch.Tensor)):
```

Comment:
Can you reorder these by most likely occurence? This will slightly speed up execution